scripts/raceEffects.config

```diff
@@ -4828,21 +4828,25 @@
 				"stat": "physicalResistance",
 				"amount": -0.2
 			},
+			{
+				"stat": "iceResistance",
+				"amount": 0.3
+			},
+			{
+				"stat": "fireResistance",
+				"amount": -0.2
+			},
 			{
 				"stat": "electricResistance",
-				"amount": 0.2
+				"amount": 0.3
 			},
 			{
 				"stat": "cosmicResistance",
 				"amount": 0.3
 			},
 			{
 				"stat": "shadowResistance",
-				"amount": -0.3
-			},
-			{
-				"stat": "radioactiveResistance",
-				"amount": 0.0

```

Comment:
Some cleanup here- removing the lines controlling the Radioactive element's resistance stat as it had no use as I had changed it to 0%.